README.md

```diff
@@ -159,46 +160,66 @@ To run e2e tests on projects affected by your changes:
 nx affected:e2e --base=origin/<base-branch>
 ```
 
-## Running The Application
+## Running The Full Stack Application
+
+Our project is a full stack application which need multiple parts to run properly. This section will detail how to run the project for different usecases.
 
-### Serving the application locally for development:
+### Serving the application locally for development
 
 1. Serve the front-end application:
    - `npm start mull-ui`
-2. Serve the back-end application (in a different tab):
+1. Serve the back-end application (in a different tab):
    - `npm start mull-api`
 
-### Building the application for production:
+### Running a local database for development
+
+You can serve the database locally for testing, and to avoid interfering with the dev/prod databases.
+
+1. Install [docker](https://docs.docker.com/get-docker/) and [MySQL Workbench](https://dev.mysql.com/downloads/workbench/) (or any similar SQL clients)
+1. Start docker
+1. Pull a `mysql` image. This image contains the information needed to spin up an instance of a mysql database using docker
+   - `docker pull mysql:5:7:32`
+1. Create and run a MySQL container, which will run an instance of MySQL
+   - `docker run --name mull-db -e MYSQL_ROOT_PASSWORD=password -p 3306:3306 -d mysql:5.7.32`
+1. Start MySQL Workbench and test the connection with your database on `localhost:3306`. If the test was successful, connect to it
+1. Update the following fields in your `.env`:
+   - `root=localhost`
+   - `password=password`, or whatever password you put for `MYSQL_ROOT_PASSWORD`
+1. Using MySQL Workbench, create a new schema your database called `mull` (case sensitive)
+
+You now should have a functioning mysql database running locally, usable for the project!
```

Comment:
Just a question, if it's a local db that will run only on your computer, why do we recommend containerizing it? 
Seems like overkill, why not just run mysql locally?
pulling in @RGPosadas @osahtout because they were working on this part

Reply:
I guess mainly because its easier if you already have docker. Instead of installing and configuring stuff manually on your computer, the docker image has all that done for you, you just need to run one or two commands

Reply:
Makes sense

Reply:
Like what @TheGreatMarkus said, it's cleaner. It would also mean you can follow these steps on different operating system